ProiectPAO/src/com/company/Classrooms.java

```diff
@@ -0,0 +1,26 @@
+package com.company;
+
+public class Classrooms extends Halls{
```

Comment:
singular noun for class names - Classroom. also, why does it extend Hall since Hall is empty?